UtopiaCity/Controllers/Sport/SportComplexController.cs

```diff
@@ -1,62 +1,80 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
 using UtopiaCity.Models.Sport;
 using UtopiaCity.Services.Sport;
+using UtopiaCity.ViewModels.Sport;
 
 namespace UtopiaCity.Controllers.Sport
 {
     public class SportComplexController : Controller
     {
         private readonly SportComplexService _sportComplexService;
+        private readonly IMapper _mapper;
 
-        public SportComplexController(SportComplexService sportComplexService)
+        public SportComplexController(SportComplexService sportComplexService, IMapper mapper)
         {
             _sportComplexService = sportComplexService;
+            _mapper = mapper;
         }
 
-        public IActionResult AllSportComplexes() => View(_sportComplexService.GetAllSportComplexes());
+        public IActionResult AllSportComplexes()
+        {
+            List<SportComplex> sportComplexes = _sportComplexService.GetAllSportComplexes();
+            List<SportComplexViewModel> sportComplexViewModels = new List<SportComplexViewModel>();
```

Comment:
just a tip - you can easily use `var` here - type is obvious from whats on the right